recipe/run_test.py

```diff
@@ -2,22 +2,30 @@
 # Distributed under the BSD-2-Clause license
 # Copyright (c) 2017, Continuum Analytics, Inc. All rights reserved.
 
-import sys
 import os
+import platform
+import sys
+
 from numba.cuda.cudadrv.libs import test, get_cudalib
 from numba.cuda.cudadrv.nvvm import NVVM
-
+from packaging import version
 
```

Comment:
Would add 2 newlines between `import`s and other code

```suggestion
from packaging import version


```

Reply:
Done